src/ol/featureloader.js

```diff
@@ -11,6 +12,16 @@ goog.require('ol.xml');
 
 
 /**
+ * {@link ol.source.Vector} sources use a function of this type to load
+ * features.
+ *
+ * This function takes an {@link ol.Extent} representing the area to be loaded,
+ * a `{number}` representing the pixel ratio and an {@link ol.proj.Projection}
```

Comment:
Minor, but I think "pixel ratio" risks being confused with `window.devicePixelRatio`.  Instead, it should read "representing the resolution (map units per pixel)" or something similar.